db/migrations/00021_bom.sql

```diff
@@ -0,0 +1,32 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE bom_info (
+  serial_num STRING PRIMARY KEY NOT NULL,
+  mac_aoc_address STRING NULL,
+  num_defi_pmi STRING NULL,
+  num_def_pwd STRING NULL,
+  metro STRING NULL
+);
+
+CREATE TABLE mac_aoc_address_bom (
```

Comment:
Minor naming quibbles: consider renaming one of the two Mac tables so they are more consistent? Either `aoc_mac_address` or `mac_bmc_address`? Also, it feels like the `_bom` suffix stutters here (i.e it repeats without adding much value).

Reply:
Done. Thanks!